zap/src/main/java/org/parosproxy/paros/view/SiteMapPanel.java

```diff
@@ -564,8 +596,8 @@ public void mouseClicked(java.awt.event.MouseEvent e) {
                         private static final long serialVersionUID = 1L;
 
                         @Override
-                        protected Context getContext() {
-                            return getSelectedContext();
+                        protected ArrayList<Context> getContexts() {
```

Comment:
This might be what's causing the build to fail, being a binary incompatible change. (As we discussed.)